activators/lex/build.gradle.kts

```diff
@@ -0,0 +1,20 @@
+import plugins.publish.POM_DESCRIPTION
+import plugins.publish.POM_NAME
+
+ext[POM_NAME] = "JAICF-Kotlin Lex Activator Adapter"
+ext[POM_DESCRIPTION] = "JAICF-Kotlin Lex Activator Adapter. Provides intent recognition and named entity extraction."
+
+plugins {
+    `jaicf-kotlin`
+    `jaicf-publish`
+    `jaicf-junit`
+}
+
+dependencies {
+    core()
+    api(jackson())
+    api("software.amazon.awssdk:bom:2.14.3")
```

Comment:
one last thing: it looks strange that bom version differs from lexruntimev2 version